Makefile

```diff
@@ -25,4 +26,8 @@ bash:
 	$(DC_RUN_PHP) bash
 style:
 	$(DC_RUN_PHP) env XDEBUG_MODE=off vendor/bin/php-cs-fixer fix --config=.php-cs-fixer.php --using-cache=no -vvv
+script:
+	ls $(SCRIPT_DIR)
```

Comment:
I'm not sure what the purpose of this is?  I didn't see `SCRIPT_DIR` defined anywhere else either.

Reply:
@tidal mentioned in our SIG meeting this was superfluous and he was going to remove it.